CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 All notable changes to this project will be documented in this file.
 
+## 2.7.6
+- [java-security] Improve compatibility of `SAPOfflineTokenServicesCloud`
+  - Now using client id (cid) from token for `clientId` attribute of `AuthorizationRequest`
```

Comment:
Hi Daniel, this is a very technical description, which is fine with me, in case you also explain the symptom here :-)